ui-v2/app/styles/components/composite-row.scss

```diff
@@ -6,7 +6,7 @@
   @extend %composite-row;
 }
 /* hoverable rows */
-.consul-upstream-list > ul > li:not(:first-child),
+.linkable,
```

Comment:
Does something like this work?

```scss
%composite-row.linkable {
  @extend %with-composite-row-intent;
}
```

?

Not sure if that works but that is what we mean here, 'any %composite-row with a class of .linkable should be a %with-composite-row-intent'.